src/server.ts

```diff
@@ -52,25 +44,22 @@ const start = async () => {
       maxAge: 7200, // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
     });
   }
-
   await registerAppPlugins(instance);
-
   // const mainMetric = SentryConfig.enable
   //   ? Sentry.startTransaction({
   //       op: 'main',
   //       name: 'Main server listen',
   //     })
   //   : null;
-
   try {
     await instance.listen({ port: PORT, host: HOSTNAME });
-    instance.log.info('App is running version %s in %s mode', APP_VERSION, ENVIRONMENT);
+    console.log('App is running version %s in %s mode', APP_VERSION, ENVIRONMENT);
```

Comment:
What are these logs?

Reply:
These logs were there already in Graasp, just to say the app is running. However, the log formattor (Pino Pretty) is not working when I deploy to AWS, so I just went back to standard console.log